Cobra-Closet/mix-match
# Haynocik/new

# Implement Cloudinary API and Styling for Wardrobe Item Cards

## Description:
This pull request introduces the complete integration of the Cloudinary API for image uploading and handles the display of wardrobe items with improved styling. Major updates include the ability to upload images to Cloudinary and retrieve URLs for display in the user's wardrobe, as well as enhanced UI components for a more intuitive and visually appealing experience.

## Key Features:
### 1. Cloudinary API Integration: 
Completed the setup for image uploading through Cloudinary. Users can now upload images of their wardrobe items, which are stored in Cloudinary, and URLs are retrieved for display.

### 2. Dynamic Item Cards in View Wardrobe: 
Implemented dynamic item cards in the view wardrobe section. These cards display item images (if available) and relevant details such as category, color, and material.

### 3. Responsive Card Styling: 
Introduced new styling for the item cards using SCSS. The cards are designed to be responsive and provide a clear and engaging user interface. Features include:

- Flip animation for cards with images.
- Neatly organized information display for easy readability.
- Uniform layout ensuring consistency across different device sizes.

### 4. CSS Enhancements: 
Added specific CSS enhancements to ensure that images fit well within the cards, maintaining aspect ratio and avoiding stretching. Margins between flex rows were adjusted for better spacing and alignment.

Bug Fixes:

- Resolved an issue where image URLs were not correctly fetching from Cloudinary.
- Fixed minor UI glitches in the item card display.

## How to Test:

- Navigate to the 'Add to Wardrobe' section and upload an image for a wardrobe item.
- Check the 'View Wardrobe' section to see the uploaded image displayed on the item card.
- Test the responsiveness of the item cards across different devices.
